app/custom_event.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>RUM Custom Event Integ Test</title>
+        <script src="./loader_standard.js"></script>
+        <link
+            rel="icon"
+            type="image/png"
+            href="https://awsmedia.s3.amazonaws.com/favicon.ico"
+        />
+
+        <script>
+            // Common to all test pages
+            function dispatch() {
+                cwr('dispatch');
+            }
+
+            // Record custom event directly using API
+            function recordEvent() {
+                cwr('recordEvent', {
+                    event_type: 'custom_event_api',
+                    event_data: { customEventVersion: 255 }
+                });
+            }
+
+            // Add plugin
+            function addPlugin() {
+                class newPlugin {
```

Comment:
Can we use js paradigm of naming classes with uppercase letters?

Reply:
Agreed!